samples/bluetooth/multiple_adv_sets/README.rst

```diff
@@ -0,0 +1,88 @@
+.. _multiple_adv_sets:
+
+Bluetooth: Multiple advertising sets
+####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Multiple advertising sets sample demonstrates how to use the Bluetooth® advertising sets.
+
+Overview
+********
+
+The sample implements two advertising sets:
+  * Non-connectable advertising with the URI to the https://www.nordicsemi.com website in the advertising data.
+    It shows the Bluetooth® Broadcaster role (*Beacon*) functionality.
```

Comment:
```suggestion
    It shows the Bluetooth Broadcaster role (*Beacon*) functionality.
```
You only need the trademark sign for the first instance of Bluetooth.

Reply:
Ok, done